src/formBuilder/utils.js

```diff
@@ -217,12 +217,6 @@ function checkObjectForUnsupportedFeatures(
   if (schema.properties)
     Object.entries(schema.properties).forEach(
       ([parameter, element]: [string, any]) => {
-        const correctName = parameter.replace(/\W/g, '_');
```

Comment:
To make sure I understand: we're no longer using any of the `parameter` values as `id`s (that's all been replaced by your `getRandomId()`), so therefore there are no longer restrictions on them?

Reply:
correct.